modules/virt-listing-vmis-web.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis-web.adoc
+
+[id="virt-listing-vmis-web_{context}"]
+= Listing standalone virtual machine instances using the web console
+
+Using the web console, you can list and view standalone virtual machine instances (VMIs) in your cluster that are not owned by virtual machines (VMs).
+
+[NOTE]
+====
+When you list VMIs using the CLI, all VMIs including standalone VMIs and VMIs that are owned by VMs and other objects display. The web console displays only standalone VMIs.
```

Comment:
How about make it more specific? Like
`The web console displays only standalone VMIs and VMs, the VMIs owned by VMs and other objects are not displayed in the list".

Reply:
I have adjusted the wording for clarity and for more specific explanation. Please review.

Reply:
Looks good.